luma/core/ansi_color.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2017 Richard Hull and contributors
+# See LICENSE.rst for details.
+
+import re
+
+valid_attributes = {
+    # Text attributes
+    0:   ["reset"],
+    7:   ["reverse_colors"],
+
+    # Foreground colors
+    30:  ["foreground_color", "black"],
+    31:  ["foreground_color", "red"],
+    32:  ["foreground_color", "green"],
+    33:  ["foreground_color", "yellow"],
+    34:  ["foreground_color", "blue"],
+    35:  ["foreground_color", "magenta"],
+    36:  ["foreground_color", "cyan"],
+    37:  ["foreground_color", "white"],
+
+    # Background colors
+    40:  ["background_color", "black"],
+    41:  ["background_color", "red"],
+    42:  ["background_color", "green"],
+    43:  ["background_color", "yellow"],
+    44:  ["background_color", "blue"],
+    45:  ["background_color", "magenta"],
+    46:  ["background_color", "cyan"],
+    47:  ["background_color", "white"],
+}
+
+
+def parse_str(text):
```

Comment:
In context of `ansi_color.parse_ansi_str`, it seems a bit of a mouthful. 
In isolation `parse_ansi_str` is probably ok, but generally: what benefit/drawback does adding 'ansi' into the method name convey?